src/main/kotlin/uk/co/craigbass/pratura/usecase/ViewBasket.kt

```diff
@@ -18,7 +18,15 @@ class ViewBasket(private val basketItemsGateway: BasketItemsRetriever,
     return PresentableBasket(lineItems(), "£${getBasketValue()}")
   }
 
-  private fun lineItems() = items!!.map { item -> PresentableLineItem(item.quantity, item.sku) }
+  private fun lineItems(): List<PresentableLineItem> {
+    return items!!.map { item ->
```

Comment:
Oh hmm sharing values between functions with nullable fields rather than parameters is a bit eyebrow raising but the scope is indeed small

Also wouldn't you still want to show individual prices against your PresentableLineItems?